inference-engine/src/transformations/src/transformations/rt_info/disable_fp16_compression.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
propose to combine these routines with `decompression*` routines in one place

Reply:
It is not a good idea in my opinion, all attribute are located in individual files, which is easier to manage